### PR TITLE
chore: do not add beacons when FORCE_PLAINTEXT_WRITE is used.

### PR DESCRIPTION
**Pull request by ajewellamz**
…h config

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: DynamoDbEncryption/dafny/DynamoDbEncryptionTransforms/test/TestFixtures.dfy [hunk 1]
```diff
@@ -232,17 +227,59 @@ module TestFixtures {
             allowedUnsignedAttributes := Some(["plain"]),
             allowedUnsignedAttributePrefix := None(),
             algorithmSuiteId := None(),
+            keyring := Some(keyring)
+          )
+        ]
+      )
+    );
+    assume {:axiom} fresh(encryption.Modifies);
```
Comment (lucasmcdonald3) [inline DynamoDbEncryption/dafny/DynamoDbEncryptionTransforms/test/TestFixtures.dfy:235]: ~~I don't know if we have team standards around `assume` statements.~~
~~An assume in a test possibly seems OK, but could you add a comment explaining why this is needed?~~

Oh, I see this is common in this file. Nevermind, but I am still curious why this assume is needed

**Review by lucasmcdonald3 — APPROVED** [review]
LGTM; is `fix` a more accurate conventional commit message than `chore`?

File: DynamoDbEncryption/dafny/DynamoDbEncryptionTransforms/src/BatchWriteItemTransform.dfy [hunk 2]
```diff
@@ -32,7 +32,7 @@ module BatchWriteItemTransform {
       //= specification/dynamodb-encryption-client/ddb-sdk-integration.md#encrypt-before-batchwriteitem
       //# If the table name does not refer to an [encrypted-table](#encrypted-table),
       //# the list of operations MUST be unchanged.
-      if tableName in config.tableEncryptionConfigs {
+      if !IsPlainWrite(config, tableName) {
```
Comment (seebees) [inline DynamoDbEncryption/dafny/DynamoDbEncryptionTransforms/src/BatchWriteItemTransform.dfy:35]: Nice use of name. I might have an early return instead to keep things flat, but

**Review by seebees — APPROVED** [review]
LGTM

**Pull request merged**